### ISSUE TITLE
No floating version dependency on svgo

### ISSUE THREAD

**Issue by simonihmig** [ivanvotti/broccoli-svg-optimizer#14]
Is there a reason why the dependency on `svgo` is locked at `0.6.6`? https://github.com/ivanvotti/broccoli-svg-optimizer/blob/master/package.json#L32

The current one is 1.1.1, which includes some new useful plugins. And I was able to use it with `ember-svg-jar` (great addon btw!), using the `svgoModule` config.

**Comment by ivanvotti** [comment]
If I remember right, when I tried to upgrade SVGO I found some breaking changes related to what plugins are active/available by default. Some existing svg-jar users would have to do changes in their configs. This test can help to see breaking changes: https://github.com/ivanvotti/broccoli-svg-optimizer/blob/master/tests/plugins-test.js

**Comment by kaungst** [comment]
Looks like there's a high vulnerability error that is fixed in svgo 1.2.2

**Comment by vertebrolog** [comment]
Are you talking about this one? https://github.com/svg/svgo/issues/395

**Comment by kaungst** [comment]
this one: https://github.com/svg/svgo/issues/1099
npm (nsp) audit fails b/c of its `js-yaml` dependency

**Comment by wagenet** [comment]
There's now a js-yaml vulnerability that the locked version of svgo uses as well.

**Comment by patrickberkeley** [comment]
@ivanvotti any thoughts on upgrading `svgo` for this repo? We are using `ember-svg-jar` (thank you for that!) as well and seeing the security vulnerabilities. Could you cut a new version of this with breaking changes and svgo updated?

**Comment by ivanvotti** [comment]
Sure, I’ll do it this week. In the mean time you can use [svgoModule](https://github.com/ivanvotti/ember-svg-jar/blob/master/docs/configuration.md#optimizer-can-be-redefined-on-a-strategy-level) option to use any svgo version:

```js
{
  svgJar: {
    optimizer : {
      svgoModule: require('svgo'),

      plugins: [
        { removeTitle: true }
      ]
    }
  }
}
```

**Comment by patrickberkeley** [comment]
Thanks @ivanvotti for cutting a new version. 

I saw the `svgoModule` option, but, correct me if I'm wrong on this, A) with npm there is no way to for version resolutions like there is with yarn, so the vulnerable version of svgo would still be installed, B) if the package is installed, the app is still vulnerable.

**Comment by ivanvotti** [comment]
I've just published `ember-svg-jar v2.0.0`, that includes `SVGO v1.3.0`. Please read [the release notes](https://github.com/ivanvotti/ember-svg-jar/blob/master/CHANGELOG.md#v200) to be ready for breaking changes.

**Comment by patrickberkeley** [comment]
🎉 Thanks @ivanvotti !
